taskcluster/app_services_taskgraph/__init__.py

```diff
@@ -82,7 +82,7 @@ def get_decision_parameters(graph_config, parameters):
         # We don't have a great way of determining if something is a nightly or
         # not.  But for now, we can assume all cron-based builds are nightlies.
         parameters["preview-build"] = "nightly"
-        parameters["target_tasks_method"] = "full"
+        parameters["target_tasks_method"] = "nightly"
```

Comment:
Might be a stupid question.. why do we set target_tasks_method here?  If we're coming from cron, target-tasks-method should already be set (in .cron.yml)?

Reply:
Not a stupid question at all, I agree that this line isn't needed.